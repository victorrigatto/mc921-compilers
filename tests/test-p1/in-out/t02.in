int main(){
    int t = .0;
    return 0;
}
