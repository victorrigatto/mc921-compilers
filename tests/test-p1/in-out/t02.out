LexToken(INT,'int',1,0)
LexToken(ID,'main',1,4)
LexToken(LPAREN,'(',1,8)
LexToken(RPAREN,')',1,9)
LexToken(LBRACE,'{',1,10)
LexToken(INT,'int',2,16)
LexToken(ID,'t',2,20)
LexToken(EQUALS,'=',2,22)
Lexical error: Illegal character '.' at 2:13
LexToken(INT_CONST,'0',2,25)
LexToken(SEMI,';',2,26)
LexToken(RETURN,'return',3,32)
LexToken(INT_CONST,'0',3,39)
LexToken(SEMI,';',3,40)
LexToken(RBRACE,'}',4,42)
